priv/templates/nimble_template/lib/otp_app_web/params/params.ex.eex

```diff
@@ -0,0 +1,23 @@
+defmodule <%= base_module %>Web.Params do
```

Comment:
can we try `<%= web_module %>` instead of `<%= base_module %>Web`

same for other places

Reply:
Updated in https://github.com/nimblehq/elixir-templates/pull/132/commits/24b9962d27018316fa95caa8e149e1d269098320